test/adapter/json_api/key_case_test.rb

```diff
@@ -36,16 +36,6 @@ class CommentSerializer < ActiveModel::Serializer
           belongs_to :author
         end
 
-        def mock_request(key_transform = nil)
-          context = Minitest::Mock.new
-          context.expect(:request_url, URI)
-          context.expect(:query_parameters, {})
-          context.expect(:key_transform, key_transform)
-          context.expect(:url_helpers, Rails.application.routes.url_helpers)
-          @options = {}
-          @options[:serialization_context] = context
-        end
```

Comment:
Yeah sure. I was a bit overwhelm afterward and didn't have time to comment on the changes.